9.4/Dockerfile

```diff
@@ -16,14 +16,25 @@ ENV POSTGRESQL_VERSION=9.4 \
     HOME=/var/lib/pgsql \
     PGUSER=postgres
 
-LABEL io.k8s.description="PostgreSQL is an advanced Object-Relational database management system" \
+ENV SUMMARY="Advanced Object-Relational database management system" \
```

Comment:
Such a sentence with a verb is not used in other containers, but in this case it sounds good to me and I have no better alternative..